src/app/pages/pages-and-posts/index.js

```diff
@@ -78,9 +79,11 @@ const PagesAndPosts = () => {
 						<ProductsPages />
 					) }
 					{ window.NewfoldRuntime.isYithBookingActive &&
-						window.NewfoldRuntime.isWoocommerceActive && (
-							<BookingAndAppointments />
-						) }
+					window.NewfoldRuntime.isWoocommerceActive ? (
+						<BookingAndAppointments />
+					) : (
+						<TransformStore />
+					) }
```

Comment:
The Jira story requirement states:

> Given customers do not have an ecommerce entitlement
When they view Pages and Posts (launchpad) in wp-admin
Then we will have CTB for ecom addon in the place of bookings card instead of removing the card altogether.

Meaning: this card (eComm addon CTB) is intended to be shown to users who do not have the e-commerce addon in their plan (ecommerce entitlement). In this condition, we are not considering whether they have the addon or not.

The condition should be:
```
IF: WooCommerce is active
AND: The user doesn't have the e-commerce addon
THEN: Show the `EcommAddonCTB` component

ELSE IF...

IF: WooCommerce is active
AND: Yith Booking is active
THEN: Show `BookingAndAppointments` component

ELSE:
Show nothing

END
```

You can check for the e-commerce entitlement using the Newfold runtime:
```js
import { NewfoldRuntime } from '@newfold-labs/wp-module-runtime';

NewfoldRuntime.hasCapability( 'isEcommerce' ) // Bool
```

Or you can:
```js
window.NewfoldRuntime.capabilities.isEcommerce // Bool | Undefined
```
Be careful with the second one because it can return `undefined`, especially in local environments or sites with connection issue.

Reply:
updated fix , but ecommerce entitlement we check with hasYithExtended.

Reply:
What's the reason for not using `isEcommerce`?

Reply:
isEcommerce refers to whether or not WooCommerce plugin is active, has nothing to do with the commerce addon.